Makefile

```diff
@@ -38,7 +38,7 @@ check_license:
 
 test:
 	@echo ">> running tests"
-	@$(GO) test -short $(pkgs)
+	@$(GO) test -race -short $(pkgs)
```

Comment:
This leads us back to the old days, where every command line thing you could ever imagine had a respective `make` target. That always seemed rather pointless to me for commands that are just as short and simple typed directly.

I see the targets more of convenience for stuff people unfamiliar with the go environment need to run and CI etc.